docs/metrics/learning-prometheus-server/README.md

```diff
@@ -0,0 +1,184 @@
+# Quick start on exporting metrics to Prometheus/Grafana
+
+- [Quick start on exporting metrics to Prometheus/Grafana](#quick-start-on-exporting-metrics-to-prometheusgrafana)
+  - [Prerequisite](#prerequisite)
+  - [Introduction](#introduction)
+  - [Get Prometheus](#get-prometheus)
+  - [Prometheus server](#prometheus-server)
+    - [Configuration](#configuration)
+    - [Start Prometheus](#start-prometheus)
+    - [Configure OpenTelemetry to Expose metrics to Prometheus Endpoint](#configure-opentelemetry-to-expose-metrics-to-prometheus-endpoint)
+    - [Check Results in Prometheus](#check-results-in-prometheus)
+    - [View/Query Results with Grafana](#viewquery-results-with-grafana)
+
+## Prerequisite
+
+It is highly recommended to go over the [getting-started](../getting-started/README.md)
+project under the metrics document folder before following along this document.
+
+## Introduction
+
+- [What is Prometheus?](https://prometheus.io/docs/introduction/overview/)
+
+- [Grafana support for
+  Prometheus](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph)
+
+## Get Prometheus
+
+Follow the [first steps]((https://prometheus.io/docs/introduction/first_steps/))
+in the prometheus official document.
+
+## Prometheus server
+
+### Configuration
+
+After downloading the [latest release](https://prometheus.io/download/), extract
+it to a local location that's easy to access. We will find the default
+Prometheus configuration YAML file in the folder, named `prometheus.yml`.
+
+Let's create a new file in the same location as `prometheus.yml`, called
+`otel.yml` for this exercise. Copy and paste the entire content below into the
+otel.yml file we have created just now.
+
+```yaml
+global:
+  scrape_interval: 10s
+  scrape_timeout: 10s
+  evaluation_interval: 10s
+scrape_configs:
+- job_name: MyOpenTelemetryDemo
+  honor_timestamps: true
+  scrape_interval: 1s
+  scrape_timeout: 1s
+  metrics_path: /metrics
+  scheme: http
+  follow_redirects: true
+  static_configs:
+  - targets:
+    - localhost:9184 # set the port that prometheus server will listen to
```

Comment:
set the target to the location where metrics will be exposed by the OpenTelemetry Prometheus Exporter